src/lib/markup/photodiode.js

```diff
@@ -1,57 +1,56 @@
-import { AT_HOME, IS_ELECTRON } from  '../../config/main'
-import { eventCodes } from '../../config/trigger'
-import $ from 'jquery'
+import { IS_ELECTRON, USE_PHOTODIODE } from "../../config/main";
+import { eventCodes } from "../../config/trigger";
+import $ from "jquery";
 
 // conditionally load electron and psiturk based on MTURK config variable
-const isElectron = IS_ELECTRON//!MTURK
+const isElectron = IS_ELECTRON; //!MTURK
```

Comment:
this variable seems redundant with `IS_ELECTRON`, just use the config constant instead